src/coreclr/nativeaot/Runtime/eventtracebase.h

```diff
@@ -103,6 +103,9 @@ struct ProfilingScanContext;
 #define CLR_MANAGEDHEAPCOLLECT_KEYWORD 0x800000
 #define CLR_GCHEAPANDTYPENAMES_KEYWORD 0x1000000
 
+// TODO: why not using the constants from ClrEtwAll.h?
```

Comment:
I don't know, but I wouldn't worry about it and follow the existing pattern

Reply:
✅ done